_pages/en/capabilities.md

```diff
@@ -535,24 +537,87 @@ InfoSec supplies development teams and operational teams with an ample number of
 
 #### 3.4.2. Indicators  
 
-- What type of tools do you use to manage the work at various stages of a process?  
-  a. Card walls  
-  b. Storyboards  
-  c. Kanban boards  
-  d. We do not use any tools  
-  e. None of the above/not applicable  
+- My team uses visual displays and dashboards that are common in the context of software delivery to manage the work at various stages of a process  
```

Comment:
- My team uses visual displays and dashboards that are common in the context of software delivery to manage the work at various stages of a process  
==>
- My team uses visual displays and dashboards to manage the work at various stages of a process